.eslintrc

```diff
@@ -27,7 +27,6 @@
     "@typescript-eslint/ban-ts-comment": ["off"],
     "@typescript-eslint/explicit-module-boundary-types": ["off"],
     "@typescript-eslint/no-explicit-any": ["off"],
-    "@typescript-eslint/no-floating-promises": ["error"],
```

Comment:
Removed this because the latest formik version makes a bunch of benign form interactions async. They did that to support react ssr/suspense.